43.md

```diff
@@ -0,0 +1,188 @@
+
+It's `.content` is set to the received "Chat Session"'s content which is enough to
+accept the session identified by the `.pubkey` and the `.content` (session privkey).
+
+## Chat Session List
+
+The [NIP-51](51.md) `kind:10043` is the user's "Chat Session List" event.
+
+It encrypts `s` (session) tags.
+
+An `s` tag's 1st value is the `pubkey` of the other side of the chat.
+The 2nd value is the session's `privkey`.
+The 3rd value is the expiration timestamp in seconds and is three weeks ahead
+of the `kind:1044`'s `.created_at` value.
+The 4th value if present is set to "1", meaning the other chat participant accepted the chat session.
+
+`s` tags of expired sessions should be kept to indicate active chats.
+
+The user should publish the event to all of their own NIP-65 `write` relays.
+
+Event example:
+
+```js
+{
+  "kind": 10043,
+  "pubkey": "<pubkey-B>"
+  "content": "<nip44EncryptV2(JSON.stringify([
+    ["s", "<pubkey-A>", "<session-privkey-A>", "1704525400", "1"],
+    // other "s" tags
+  ]))>",
+  // ...other fields
+}
+```
+
+## Session Channel
+
+The session channel is simply made of the set of NIP-59 `kind:1059` events
+with the session pubkey as their author.
+
+Either the sender or the recipient may delete these events because both of them know
+the session's privkey. Therefore, clients should store them locally to keep their users in control
+of the DMs lifecycle.
+
+**Important**: The NIP-44 encryption when wrapping and when sealing session channel events
+uses the first 23 characters of the session's `privkey` as salt suffix.
+This ensures forward secrecy when creating a new session because both users
+are supposed to store just the last session's `privkey`
+in the "Chat Session List" event.
+
+### DM
+
+The DM is a `kind:14` NIP-59 rumor.
+
+This event should be sent to atleast one of the recipient's NIP-65 `read` relays
+and to atleast one of the sender's `read` relays.
+This way, an user will only have to fetch DMs from their own set of `read` relays.
+
+DM event example:
+
+```js
+{
+  "kind": 1059, // gift wrap
+  "pubkey": "<session-pubkey-A>",
+  "tags": [],
+  "content": "<nip44EncryptV2(JSON.stringify({
+    "kind": 13, // seal
+    "pubkey" "<pubkey-A>",
+    "tags": [],
+    "content": "<nip44EncryptV2(JSON.stringify({
+      "id": "<...>",
+      "kind": 14, // DM
+      "pubkey" "<pubkey-A>",
+      "tags": [],
+      "content": "Hello, User B",
+      "created_at": 1702711000 // now
+      // no .sig field
+    }))>",
+    "created_at": 0
+    // ....other fields
+  }))>",
+  "created_at": 1702711000 // now
+  // ...other fields
+}
+```
+
+### Last Received At
+
+Optionally, the `created_at` of a `kind:15` NIP-59 rumor
+informs the last time an user's client received DMs from the other participant.
+
+This event should be sent to atleast one of the recipient's NIP-65 `read` relays.
+
+### Last Read At
+
+Optionally, the `created_at` of a `kind:16` NIP-59 rumor
```

Comment:
Nit: `kind:16` is a `generic repost` defined in NIP-18.

Reply:
fixed. thanks!